sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryStatus.scala

```diff
@@ -28,9 +28,10 @@ import org.apache.spark.annotation.InterfaceStability
  * Reports information about the instantaneous status of a streaming query.
  *
  * @param message A human readable description of what the stream is currently doing.
- * @param isDataAvailable True when there is new data to be processed.
+ * @param isDataAvailable True when there is new data to be processed. Doesn't apply
+ *                        to ContinuousExecution and always false.
```

Comment:
"and always false" => where it is always false. 
(Same for a few lines below)

Reply:
Changed.